_includes/v23.1/performance/reduce-hot-spots.md

```diff
@@ -0,0 +1,31 @@
+- Use index keys with a random distribution of values, so that transactions over different rows are more likely to operate on separate data ranges. See the [SQL FAQs](sql-faqs.html#how-do-i-auto-generate-unique-row-ids-in-cockroachdb) on row IDs for suggestions.
+
+- Place parts of the records that are modified by different transactions in different tables. That is, increase [normalization](https://en.wikipedia.org/wiki/Database_normalization). However, there are benefits and drawbacks to increasing normalization.
+
+    - Benefits:
+
+        - Allows separate transactions to modify related underlying data without causing [contention](#transaction-contention).
+        - Can improve performance for read-heavy workloads.
+
+    - Drawbacks:
+
+        - More complex data model.
+        - Increases the chance of data inconsistency.
+        - Increases data redundancy.
+        - Can degrade performance for write-heavy workloads.
+
+- If the application strictly requires operating on very few different index keys, consider using [`ALTER ... SPLIT AT`](alter-table.html#split-at) so that each index key can be served by a separate group of nodes in the cluster.
+
+- If you are working with a table that **must** be indexed on sequential keys, consider using [hash-sharded indexes](hash-sharded-indexes.html). For details about the mechanics and performance improvements of hash-sharded indexes in CockroachDB, see the blog post [Hash Sharded Indexes Unlock Linear Scaling for Sequential Workloads](https://www.cockroachlabs.com/blog/hash-sharded-indexes-unlock-linear-scaling-for-sequential-workloads/). As part of this, we recommend doing thorough performance testing with and without hash-sharded indexes to see which works best for your application.
+
+- To avoid read hot spots:
+
+    - Increase data distribution, which will allow for more ranges. The hot spot exists because the data being accessed is all co-located in one range.
+    - Increase [load balancing](recommended-production-settings.html#load-balancing) across more nodes in the same range. Most transactional reads must go to the leaseholder in CockroachDB, which means that opportunities for load balancing over replicas are minimal.
+
+        However, the following features do not permit load balancing over replicas:
```

Comment:
I assume this is a typo and should say that the following features **do** permit load balancing over replicas?